app/lib/features/pins/pages/pin_details_page.dart

```diff
@@ -75,49 +76,83 @@ class _PinDetailsPageState extends ConsumerState<PinDetailsPage> {
           ),
         );
       },
-      loading: () => Skeletonizer(
-        child: Text(L10n.of(context).loadingPin),
-      ),
-      error: (e, s) {
-        _log.severe('Error loading pin', e, s);
-        return Text(
-          L10n.of(context).errorLoadingPin(e),
+      loading: _contentLoader,
+      error: (error, stack) {
+        _log.severe('Error loading pin', error, stack);
+        return ErrorPage(
+          background: _contentLoader(),
```

Comment:
Ahh, now I got it. Show we are showing some good loading UI when showing error info.

Reply:
Yes, it's the background animation.